apps/google-analytics-4/frontend/src/components/main-app/AnalyticsMetricDisplays/AnalyticsMetricDisplay.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ChartFooter from 'components/main-app/ChartFooter/ChartFooter';
+import ChartHeader from 'components/main-app/ChartHeader/ChartHeader';
+import ChartContent from '../ChartContent/ChartContent';
+import { RunReportResponse } from 'types';
+
+interface Props {
+  handleDateRangeChange: Function;
+  runReportResponse: RunReportResponse;
+  reportSlug: string;
+  pageViews: number;
+  metricName: string;
+  error?: Error;
+}
+
+const AnalyticsMetricDisplay = (props: Props) => {
+  const { handleDateRangeChange, runReportResponse, reportSlug, error, metricName, pageViews } =
+    props;
+  return (
+    <>
+      <ChartHeader
+        metricName={metricName ? metricName : ''}
+        metricValue={pageViews || pageViews === 0 ? pageViews.toString() : ''}
+        handleChange={handleDateRangeChange}
+      />
+
+      <ChartContent error={error} pageViewData={runReportResponse} />
+
+      <ChartFooter
+        slugName={`Page path: ${reportSlug}` || ''}
```

Comment:
If I'm understanding the sentiment of this correctly, did you mean to do: ```reportSlug ? `Page path: ${reportSlug}` || ''```?

Reply:
ah yes this is actually leftover code from pervious PR's that I just moved to this file, so not a direct change but I will adjust to enforce the conditional logic. Great catch!